### PR TITLE
Feature Branch for new button and such

### PR DESCRIPTION
**Pull request by n00bcake**
also gave the little dude some poi and (hopefully) made it workable on older OS's. 

This has been an unexpected long and arduous road, challenging my visual learning mind with a new style. I have learned so much. Thank you for the time you're taking to help me with this.
Hopefully in the future it will serve the world of poi well and I will find a way to pass the community's collective thanks your way. Thank you!

### CODE REVIEW

**Comment by Mitchlol** [comment]
You pulled to the wrong branch :D Pull request needs to be to develop :)

File: app/build.gradle [hunk 1]
```diff
@@ -6,7 +6,7 @@ android {
 
     defaultConfig {
         applicationId "com.mitchelllustig.spinmorepoi"
-        minSdkVersion 16
+        minSdkVersion 10
```
Comment (Mitchlol) [inline app/build.gradle]: This change should be as a separate feature, easist way is just change it back as another commit to this feature, then re-make the changes on a new feature branch

If your feature name has the word "and" in it, your probably doing it wrong :-1:

File: app/src/main/java/com/mitchelllustig/spinmorepoi/RandomizerActivity.java [hunk 2]
```diff
@@ -41,4 +41,11 @@ public void playRandomVideo(){
...
         }
     }
+
+    //Here's my shitty attempt to simply make a button open up a url in a browser
+    @OnClick(R.id.button2)
+    public void openWebURL() {
+        Intent browse = new Intent( Intent.ACTION_VIEW , Uri.parse( "http://www.spinmorepoi.com/" ) );
+        startActivity( browse );
+    }
 }
```
Comment (Mitchlol) [inline app/src/main/java/com/mitchelllustig/spinmorepoi/RandomizerActivity.java:51]: This looks good :+1:

File: app/src/main/res/layout/randomizer_layout.xml [hunk 3]
```diff
@@ -9,15 +9,47 @@
...
+        android:textSize="20dp"
         android:padding="20dip"
-        android:layout_margin="10dip"/>
+        android:layout_alignParentBottom="true"
+        android:layout_centerHorizontal="true"
+        android:layout_marginBottom="47dp"
+        android:clickable="true" />
 
-</RelativeLayout>
+</RelativeLayout>
```
Comment (Mitchlol) [inline app/src/main/res/layout/randomizer_layout.xml]: All files should have a blank like at the end of the file. (don't ask, its a thing, you can see the red error on the line here)

File: app/src/main/res/layout/randomizer_layout.xml [hunk 4]
```diff
@@ -9,15 +9,47 @@
...
+        android:gravity="center"
         android:background="@drawable/smp_sign"
-        android:text="Click here for a random poi video!"
+        android:text="Press for a random poi video."
         android:textColor="@drawable/csl_light"
         android:textSize="50dp"
+        android:padding="20dip"
+        android:layout_margin="10dip"
         android:layout_centerInParent="true"
+        android:nestedScrollingEnabled="false"
```
Comment (Mitchlol) [inline app/src/main/res/layout/randomizer_layout.xml:39]: Why is this line here?

File: app/src/main/res/layout/randomizer_layout.xml [hunk 5]
```diff
@@ -9,15 +9,47 @@
...
+        android:clickable="false" />
```
Comment (Mitchlol) [inline app/src/main/res/layout/randomizer_layout.xml]: And this line as well?

File: app/src/main/res/layout/randomizer_layout.xml [hunk 6]
```diff
@@ -9,15 +9,47 @@
...
+        android:layout_marginTop="300dp"/>
     <TextView android:id="@+id/button"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
+        android:gravity="center"
         android:background="@drawable/smp_sign"
-        android:text="Click here for a random poi video!"
+        android:text="Press for a random poi video."
         android:textColor="@drawable/csl_light"
         android:textSize="50dp"
```
Comment (Mitchlol) [inline app/src/main/res/layout/randomizer_layout.xml]: Try making this text size 40dp, and see if u like the look better

File: app/src/main/res/layout/randomizer_layout.xml [hunk 7]
```diff
@@ -9,15 +9,47 @@
         android:layout_height="match_parent"
         android:src="@drawable/smp_dude"
         android:padding="15dip"/>
+    <ImageView
```
Comment (Mitchlol) [inline app/src/main/res/layout/randomizer_layout.xml:12]: How can poi guy hold both poi, and sign? Up to you tho :-P

File: app/src/main/res/layout/randomizer_layout.xml [hunk 8]
```diff
@@ -9,15 +9,47 @@
...
+        android:text="Press for a random poi video."
         android:textColor="@drawable/csl_light"
         android:textSize="50dp"
+        android:padding="20dip"
+        android:layout_margin="10dip"
         android:layout_centerInParent="true"
+        android:nestedScrollingEnabled="false"
+        android:clickable="false" />
+
+    <TextView android:id="@+id/button2"
```
Comment (Mitchlol) [inline app/src/main/res/layout/randomizer_layout.xml:42]: Maybe this look better if he stand on top of it?

File: app/src/main/res/layout/randomizer_layout.xml (hunk 4 above)
Comment (n00bcake) [inline app/src/main/res/layout/randomizer_layout.xml:39]: I did not add this. I swear. My computer seemed to make tons of changes on its own, which is why I only pushed these 3 files.

File: app/src/main/res/layout/randomizer_layout.xml (hunk 5 above)
Comment (n00bcake) [inline app/src/main/res/layout/randomizer_layout.xml]: ditto

File: app/src/main/res/layout/randomizer_layout.xml [hunk 9]
```diff
@@ -9,15 +9,47 @@
...
-        android:text="Click here for a random poi video!"
+        android:text="Press for a random poi video."
         android:textColor="@drawable/csl_light"
-        android:textSize="50dp"
+        android:textSize="40dp"
+        android:padding="20dip"
+        android:layout_margin="10dip"
         android:layout_centerInParent="true"
+        android:nestedScrollingEnabled="false"
+        android:clickable="false" />
```
Comment (Mitchlol) [inline app/src/main/res/layout/randomizer_layout.xml:40]: Why are these two lines added?

android:nestedScrollingEnabled="false"
android:clickable="false"
